### ISSUE TITLE
I forked the project as Breezy Weather

### ISSUE THREAD

**Issue by papjul** [WangDaYeeeeee/GeometricWeather#492]
TL;DR: I forked Geometric Weather under the name [Breezy Weather here](https://github.com/breezy-weather/breezy-weather). Many features were added as well as tons of fixes. Code was modernized and translations updated.

___

First of all, a kind word for @WangDaYeeeeee: I and many others here are extremely grateful for this wonderful app you developed for years. We barely see any activity from you anymore and I can only guess that life moved on for you.

Given my previous experiences on this project that I like a lot, I decided to give it a try and maintain it myself. Currently, the whole code was rewritten to Kotlin to ease future maintenance, all libs were updated and a GitHub bot is ensuring we always have latest version, translation is done from Weblate which ease the work of contributors. Finally, a GitHub CI was implemented, which automate GitHub releases so there is almost nothing to do anymore to publish a new release.

So, if you got bored or lacked time for Geometric Weather because of these previous points, know that they are now solved and you will always feel welcome in the GitHub organization if you want to focus on what you do the best and not the maintenance part. Otherwise, I wish you success for the future!

(Note that if you don’t plan to do anything to your repo anymore, I would suggest linking to Breezy Weather in your README and archive the project, so users don’t keep opening new issues and think project is active)

___

For those of you who don't know me, a bit of history:
@mbestavros initially made a fork under Geometric Weather FOSS name, changing application ID, of Geometric Weather to remove proprietary dependencies. It was [added to F-Droid](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/7315).

Then, [I made one of my first pull requests](https://github.com/WangDaYeeeeee/GeometricWeather/pull/180) to make a variant on the official repo which allowed to build a specific variant for F-Droid, so the [official app was merged again](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8418) into F-Droid with the original applicationId this time.

I continued making [many contributions](https://github.com/WangDaYeeeeee/GeometricWeather/pulls?q=is%3Apr+author%3Apapjul+) accepted upstream.

It now has been more than a year since latest beta, and despite many issues opened and emails sent, activity was nearly inexistant.

Given I did not have rights to the official repo, I had to fork the app to be able to make new releases.

There are now many new features, technical improvements and bug fixes I can’t list all of them. First alpha was released 2 weeks ago and tested by many. As of writing, we are in beta although I consider the app already very stable (given regressions and small bugs are almost fixed in a blink), probably more than latest Geometric Weather given it fixes many of its bugs.

You can download app from [IzzyOnDroid repo ](https://apt.izzysoft.de/fdroid/index/apk/org.breezyweather/) or from [GitHub releases](https://github.com/breezy-weather/breezy-weather/releases) (for example, [through Obtainium](https://github.com/ImranR98/Obtainium)).

If you have questions or issues to report, please use GitHub discussions or issues from Breezy Weather repo instead of replying here, thank you!

**Comment by CelianGdfrd** [comment]
Thank you for doing this, mad props and respect to @WangDaYeeeeee for making such an amazing libre app

**Comment by ghobs91** [comment]
Awesome work! Are there plans for this fork to have the iOS version on TestFlight, like Geometric Weather does?

**Comment by papjul** [comment]
> Awesome work! Are there plans for this fork to have the iOS version on TestFlight, like Geometric Weather does?

iOS version is a different app, you're looking for this repo: https://github.com/WangDaYeeeeee/GeometricWeather-iOS
